src/proto/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# (C) Copyright 2019 Intel Corporation.
+# (C) Copyright 2020 Intel Corporation.
```

Comment:
Nit: Copyright updates are supposed to be a range, aren't they? i.e. 2019-2020, since the file was created in 2019 and updated in 2020.